docs/API.ReturnCodes.md

```diff
@@ -9,5 +9,6 @@ Return Code | Description
 204 | No more results
 404 | Infoton not found
 401 | Unauthorized user, or password does not match user name
+422 | The sub-graph you requested to delete does not exist
```

Comment:
It's not about non existent sub graphs. Every request that is semantically correct, but has no effect or meaning, should result in 422. Another example, is ingesting something like:
```
<http://my.domain.com/some/path> <cmwell://meta/sys#path> "/my.domain.com/some/path" .
```